docs/API.md

```diff
@@ -43,6 +43,7 @@ Takes the GraphQL resolveInfo and returns a hydrated Object with the data.
 | dbCall | [<code>dbCall</code>](#dbCall) | A function that is passed the compiled SQL that calls the database and returns a promise of the data. |
 | [options] | <code>Object</code> |  |
 | options.minify | <code>Boolean</code> | Generate minimum-length column names in the results table. |
+| options.prefixAliases | <code>Boolean</code> | Prefix column and table names with "$" to avoid conflicts in subquery expressions. |
```

Comment:
I would make this a string to make it more flexible. Imagine the user relies on a system that generates subqueries for them and that subsystem uses "$" as prefix, we will end up in the same situation :O